go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/go-jet/jet/v2
 
-go 1.18
+go 1.22.0
```

Comment:
Why is it necessary to increase the minimum go version?

Reply:
Let me try to bump that down, there is a pattern where setup-go can simply pull the version from the go.mod which is convenient, but not having to force everyone to upgrade is valid too.  Let me see if this works with 1.18.

Reply:
Yup, this works.  I think it got updated when did a go install of gosec.  Reverted.